duckduckgo/Android
# WebView Dark theme support

Task/Issue URL: https://app.asana.com/0/414730916066338/1174433894299360/f
Tech Design URL: 
CC: 

**Description**:
This PR introduces new flags on the webview configuration to automatically use website dark them if DDG browser theme is Dark.


**Steps to test this PR**:
- Test:
1. Install the app
2. Visit SERP page
1. SERP should render in light mode
2. Change the app to use Dark theme
3. Go back to browser
4. Ensure SERP renders in dark mode
5. Go to SERP settings
6. Ensure dark theme is selected
7. Change the theme in SERP settings
8. Ensure selected theme is honored by webview
9. Now visit Twitter
10. Ensure it renders in dark mode
11. Change the app them to light
12. Ensure Twitter now is rendered in light theme


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
